nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/drive/ListGoogleDrive.java

```diff
@@ -194,7 +194,7 @@ public void onScheduled(final ProcessContext context) throws IOException {
 
         HttpTransport httpTransport = new ProxyAwareTransportFactory(proxyConfiguration).create();
 
-        driveService = createDriveService(context, httpTransport, DriveScopes.DRIVE_METADATA_READONLY);
```

Comment:
I think this change is unnecessary. The `DRIVE_METADATA_READONLY` scope should be enough for the List processor.